client/components/editor/index.vue

```diff
@@ -153,10 +174,25 @@ export default {
     if (!this.course) actions.push(this.getCourse(courseId));
     Promise.all(actions).then(() => (this.showLoader = false));
   },
+  async mounted() {
```

Comment:
I noticed that this code is pretty much the same as in ```Outline``` component so you should move to to a mixin